tests/user/test_user_debug_logging.py

```diff
@@ -184,6 +187,88 @@ def test_add_user_request_debug_log_passphrases_and_passwords_get_redacted_on_er
         self.assertNotIn(self.test_passphrase, error_log)
         self.assertNotIn(self.test_password, error_log)
 
+    def test_add_user_request_debug_log_password_xml_tags_not_redacted_on_success(
+        self,
+        call_racf_mock: Mock,
+    ):
+        call_racf_mock.return_value = (
+            TestUserConstants.TEST_ADD_USER_PASSWORD_RESULT_SUCCESS_XML
+        )
+        stdout = io.StringIO()
+        with contextlib.redirect_stdout(stdout):
+            self.user_admin.add(
+                "squidwrd",
+                traits=TestUserConstants.TEST_ADD_USER_REQUEST_TRAITS_PASSWORD_SIMPLE,
+            )
+        success_log = self.ansi_escape.sub("", stdout.getvalue())
+        self.assertEqual(
+            success_log, TestUserConstants.TEST_ADD_USER_PASSWORD_SUCCESS_LOG
+        )
+        self.assertEqual(success_log.count("********"), 4)
+        self.assertIn(self.simple_password, success_log)
+
+    def test_add_user_request_debug_log_password_xml_tags_not_redacted_on_error(
+        self,
+        call_racf_mock: Mock,
+    ):
+        call_racf_mock.return_value = (
+            TestUserConstants.TEST_ADD_USER_PASSWORD_RESULT_ERROR_XML
+        )
+        stdout = io.StringIO()
+        with contextlib.redirect_stdout(stdout):
+            try:
+                self.user_admin.add(
+                    "squidwrd",
+                    traits=TestUserConstants.TEST_ADD_USER_REQUEST_TRAITS_PASSWORD_SIMPLE,
+                )
+            except SecurityRequestError:
+                pass
+        error_log = self.ansi_escape.sub("", stdout.getvalue())
+        self.assertEqual(error_log, TestUserConstants.TEST_ADD_USER_PASSWORD_ERROR_LOG)
+        self.assertEqual(error_log.count("********"), 4)
+        self.assertIn(self.simple_password, error_log)
+
+    # ============================================================================
+    # Add Additional Secrets
+    # ============================================================================
+    def test_add_user_request_debug_log_additional_secret_added_get_redacted_on_success(
+        self,
+        call_racf_mock: Mock,
+    ):
+        user_admin = UserAdmin(debug=True, additional_secret_traits=["omvs:uid"])
+        call_racf_mock.return_value = TestUserConstants.TEST_ADD_USER_RESULT_SUCCESS_XML
+        stdout = io.StringIO()
+        with contextlib.redirect_stdout(stdout):
+            user_admin.add(
+                "squidwrd",
+                traits=TestUserConstants.TEST_ADD_USER_REQUEST_TRAITS,
+            )
+        success_log = self.ansi_escape.sub("", stdout.getvalue())
+        self.assertEqual(
+            success_log,
+            TestUserConstants.TEST_ADD_USER_ADDITIONAL_SECRET_ADDED_SUCCESS_LOG,
+        )
```

Comment:
I think it probably makes sense to do an `assertNotIn` here for the additional secrets.

Reply:
Added the following for these tests:

```python
self.assertNotIn(TestUserConstants.TEST_ADD_USER_REQUEST_TRAITS["omvs:uid"], success_log)
```